networks/suzuka/suzuka-full-node/src/manager.rs

```diff
@@ -27,6 +29,25 @@ impl Manager<SuzukaPartialNode<Executor>> {
     }
 
     pub async fn try_run(&self) -> Result<(), anyhow::Error> {
+
+        let (stop_tx, mut stop_rx) = tokio::sync::watch::channel(());
+        tokio::spawn({
+            let mut sigterm = signal(SignalKind::terminate()).context("Can't register to SIGTERM.")?;
+            let mut sigint = signal(SignalKind::interrupt()).context("Can't register to SIGKILL.")?;
```

Comment:
```suggestion
            let mut sigint = signal(SignalKind::interrupt()).context("Can't register to SIGINT.")?;
```

Reply:
This is resolved in https://github.com/movementlabsxyz/movement/pull/191. I'll leave it to that.